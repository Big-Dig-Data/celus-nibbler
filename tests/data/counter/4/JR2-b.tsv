Journal	Publisher	Platform	Journal DOI	Proprietary Identifier	Print ISSN	Online ISSN	Access Denied Category	Reporting Period Total	Mar-2022
Total for all journals	Annual Reviews	Annual Reviews					Access denied: content item not licensed	7	7
Journal1	Annual Reviews	Annual Reviews	10.1111/journal1	economics	1111-1111	2222-2222	Access denied: content item not licensed	2	2
Journal2	Annual Reviews	Annual Reviews	10.2222/journal2	financial	3333-3333	4444-4444	Access denied: content item not licensed	4	4
Journal3	Annual Reviews	Annual Reviews	10.3333/journal3	resource	6666-6666	5555-5555	Access denied: content item not licensed	1	1
