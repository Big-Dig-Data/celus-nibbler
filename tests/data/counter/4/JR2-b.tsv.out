JR2-a.tsv.out